### PR TITLE
Lxd fixes

### PR DESCRIPTION
**Pull request by tych0**
(Review request: http://reviews.vapour.ws/r/4598/)

### CODE REVIEW

File: tools/lxdclient/lxd_client.go [hunk 1]
```diff
@@ -27,6 +27,8 @@ const (
 	StatusCancelled        = "Canceled"
 	StatusSuccess          = "Success"
 	StatusFailure          = "Failure"
+
+	DefaultLXDBridge = "lxdbr0"
```
Comment (frobware) [inline tools/lxdclient/lxd_client.go]: Alignment; go fmt?

Comment (tych0) [inline tools/lxdclient/lxd_client.go]: We don't need it because of the \n.

File: worker/provisioner/lxd-broker.go [hunk 2]
```diff
@@ -54,13 +54,9 @@ func (broker *lxdBroker) StartInstance(args environs.StartInstanceParams) (*envi
 		return nil, errors.New("starting lxd containers with networks is not supported yet")
 	}
 	machineId := args.InstanceConfig.MachineId
-	bridgeDevice := broker.agentConfig.Value(agent.LxcBridge)
+	bridgeDevice := broker.agentConfig.Value(agent.LxdBridge)
```
Comment (dimitern) [inline worker/provisioner/lxd-broker.go]: FWIW, as discussed on IRC - agent.LxcBridge is always empty, as it's not getting populated by providercommon.Bootstrap() used by most providers. So the `if` below is always going to succeed.

**Comment by tych0** [comment]
This is against the wrong branch.

**Pull request closed without merging**